phpBB/includes/acp/acp_board.php

```diff
@@ -530,11 +530,13 @@ function main($id, $mode)
 
 			if ($submit)
 			{
-				if (strpos($data['type'], 'password') === 0 && $config_value === '********')
+				if (strpos(isset($data['type']) ? $data['type'] : '', 'password') === 0 && $config_value === '********')
```

Comment:
Could we not move the `isset()` outside of the `strpos()`?
As if it is not set, we don't have bother with rest anyways.

Reply:
Also doable yes. @marc1706 ?

Reply:
Yes, would make more sense.